set.mm

```diff
@@ -504787,7 +504923,7 @@ required as (atomic) formulas, can be introduced as a defined notion in
      formulas, see ~ fmla .  (Contributed by Mario Carneiro, 14-Jul-2013.) $)
   df-fmla $a |- Fmla = ( n e. suc _om |-> dom ( ( (/) Sat (/) ) ` n ) ) $.
 
-$( --- theorems for ` ( I e.g J ) ` and ` A.g M A ` --- $)
+$( --- theorems for ` ( I e.g J ) `, ` ( A |g B ) ` and ` A.g M A ` --- $)
```

Comment:
I see you're only updating a pre-existing comment.
Did you consider turning this into a (sub-sub)-section?

Reply:
Since I am working within MC's mathbox in this case, I do not want to change the structure of it. This should be decided by @digama0. Or the whole section should be moved to main...

Reply:
> Or the whole section should be moved to main...

Makes sense to me. Especially once we have multiple people working on it, keeping it in a mathbox doesn't seem all that appealing.

Reply:
So how/where should it be placed in main? I think a new "part" (as PART 17, following PART 16 GRAPH THEORY) is required for it, which could be titled "MODEL THEORY" (according to @digama0's comment in #3572). Its (currently one and only) section would be "Gödel-sets of formulas", containing the subsections "Primary Gödel-sets", "The Satisfaction predicate", "Gödel formulas", "The "proves" relation", "Additional Gödel-sets".

Additional candidates for this new PART would be the following sections in MC's mathbox:

- 20.6.12  Models of ZF  
- 20.6.13  Metamath formal systems  
- 20.6.14  Grammatical formal systems  
- 20.6.15  Models of formal systems  

I also detected section "Provability logic" in BJ's mathbox, where a "provability predicate" is defined and theorems for Gödel's second incompleteness theorem ( ~ bj-babygodel ) and Löb's  theorem ( ~ bj-babylob ) are provided. Would this also fit into this new PART?

@benjub @digama0 @david-a-wheeler what is your opinion?

I will open a new PR for a correspondig proposal, if nobody objects.

Reply:
@avekens can you open an issue for this ? (you can simply paste your text there, as a starting point).

Reply:
Issue #3599 is opened to discuss this (and finally to make a decision).